app/main/routes_frontend.py

```diff
@@ -197,6 +197,35 @@ def get_pico_recipes():
 
 def load_active_brew_sessions():

```

Comment:
break this method into 2:
- [] one purposed to initialize all sessions from disk 
- [] other for loading the index page of active sessions from memory.

Reply:
split into `load_active_brew_sessions` and `restore_active_sessions` functions